### PR TITLE
feature/status-toelichting-es

### PR DESCRIPTION
**Pull request by damm89**

### CODE REVIEW

**Review by sergei-maertens — APPROVED** [review]

File: backend/src/zac/elasticsearch/api.py [hunk 1]
```diff
@@ -24,11 +32,23 @@ def create_zaak_document(zaak: Zaak) -> ZaakDocument:
         if isinstance(zaak.zaaktype, ZaakType)
         else fetch_zaaktype(zaak.zaaktype)
     )
+    zaak.zaaktype = zaaktype
```
Comment (annashamray) [inline backend/src/zac/elasticsearch/api.py:35]: Looks like this line is excessive

File: backend/src/zac/elasticsearch/api.py [hunk 2]
```diff
@@ -44,20 +64,33 @@ def create_zaak_document(zaak: Zaak) -> ZaakDocument:
...
+        status=status_document,
+        toelichting=zaak.toelichting,
     )
     zaak_document.save()
     # TODO check rollen in case of update
     return zaak_document
 
 
-def update_zaak_document(zaak: Zaak) -> ZaakDocument:
+def _get_zaak_document(
```
Comment (annashamray) [inline backend/src/zac/elasticsearch/api.py:75]: Nice DRYing the code :)

File: backend/src/zac/elasticsearch/api.py [hunk 3]
```diff
@@ -142,3 +166,30 @@ def update_eigenschappen_in_zaak_document(zaak: Zaak):
 
     zaak_document.eigenschappen = eigenschappen_doc
     zaak_document.save()
+
+    return
+
+
+def _create_status_document(zaak: Zaak) -> StatusDocument:
```
Comment (annashamray) [inline backend/src/zac/elasticsearch/api.py:173]: I think this function can also be reused inside `create_zaak_document` function

**Review by annashamray — APPROVED** [review]
Nice and neat!

**Comment by annashamray** [comment]
@damm89 if there is some issue for this feature link it with this PR please

**Comment by damm89** [comment]
Related to https://github.com/GemeenteUtrecht/ZGW/issues/851

**Comment by annashamray** [comment]
yes, it's added to the endpoint `/api/workstack/cases`

**Pull request merged**